build/testRunTimeImages.sh

```diff
@@ -23,6 +23,20 @@ else
     $buildRuntimeImagesScript "$@"
 fi
 
+if [ -n "$2" ]
+then
+    echo
+    echo "Setting environment variable 'ORYX_TEST_IMAGE_BASE' to provided value '$2'."
+    export ORYX_TEST_IMAGE_BASE="$2"
```

Comment:
Same here: `ORYX_TEST_REGISTRY_NAME` and change the name of the other down below too

Reply:
Changed this environment variable.